spark-jobs/src/main/resources/reference.conf

```diff
@@ -25,10 +25,11 @@
 menas.rest.uri="http://localhost:8080,host2:9000/menas;https://localhost:8080,host2:9000/menas"
 # Each of the above uris can be tried multiple times for fault-tolerance (optional)
 menas.rest.retryCount=0
+menas.rest.optionallyRetryableExceptions=[401,403,404]
+
```

Comment:
This will change the behavior of Enceladus deployment. Keep it empty and only in comment list the supported ones.

Reply:
Btw, what's the behavior if the key is not present at all? Ideally should default to empty list (not keep compatibility. But if in `reference.conf` still OK.

Reply:
Very good point. I think that we have to make a decision about whether this config should be mandatory or not.

I can see that `menas.rest.retryCount` is optional, which might indicate that the whole 'retry mechanism' - driven by the configuration - is 'optional'. Can I change this in a way that if this is empty/not existing it will return an empty List?

(I didn't want to make this decision alone since I just joined the team/project.)